### ISSUE TITLE
Provider "fortimanager" / Admin Sessions for JSON do not logout properly

### ISSUE THREAD

**Issue by fortijames** [fortinetdev/terraform-provider-fortimanager#7]
When doing a terraform plan, for example, the code starts a session and obtains a new sessions ID. It never does a logout and the sessions pile up. 

This is after doing a `terraform plan` x3 without any changes. Two of these sessions are expected (GUI and SSH) but the JSON sessions should be getting cleared from the terraform API client. 
```
FMG1 # diagnose system admin-session list
*** entry 0 ***
session_id: 35714 (seq: 0)
username: jim
admin template: jim
from: GUI(192.168.8.1) (type 1)
profile: Super_User (type 3)
adom: root
session length: 967 (seconds)
idle: 60 (seconds)

*** entry 1 ***
session_id: 63377 (seq: 0)
username: admin
admin template: admin
from: ssh(192.168.0.10) (type 0)
profile: Super_User (type 3)
adom: root
session length: 338 (seconds)

*** entry 2 ***
session_id: 47917 (seq: 1)
username: admin
admin template: admin
from: JSON(192.168.0.10) (type 3)
profile: Super_User (type 3)
adom: root
session length: 241 (seconds)

*** entry 3 ***
session_id: 10498 (seq: 2)
username: admin
admin template: admin
from: JSON(192.168.0.10) (type 3)
profile: Super_User (type 3)
adom: root
session length: 237 (seconds)

*** entry 4 ***
session_id: 15401 (seq: 3)
username: admin
admin template: admin
from: JSON(192.168.0.10) (type 3)
profile: Super_User (type 3)
adom: root
session length: 234 (seconds)
FMG1 # 
```

**Comment by lix-fortinet** [comment]
Hi @fortijames,

Thank you for raising this issue. Team are working on this issue. We will reply you ASAP.

Thanks,
Xing

**Comment by nikitakazantsev12** [comment]
I confirm the same issue still exists in the latest provider version (1.7.0) and has not been addressed yet. The Terraform operations stop as soon as the sessions limit is reached, and can be continued only after those sessions are being automatically killed after the idle time has passed. The session closure is a *major* issue in the provider.

**Comment by lix-fortinet** [comment]
Hi @fortijames, @nikitakazantsev12,

The latest release of FortiManager Terraform provider v1.8.0 support token based authentication, which do not have the session limit issue. Here is the reference: https://registry.terraform.io/providers/fortinetdev/fortimanager/latest/docs/guides/fmg_token. Please try it out and let me know if you have any questions.

Thanks,
Xing

**Comment by lix-fortinet** [comment]
Hi,

We will close this issue since it has been fixed. Feel free to open a new issue if you have any questions or issues.

Thanks,
Xing
